src/components/custom/other-platforms.js

```diff
@@ -341,6 +343,18 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
                     onClick={onClick}
                     otherLinkProps={{ rel: 'noopener noreferrer' }}
                 />
+                {is_eu_country && (
```

Comment:
Do we need to duplicate this code for EU and non_eu accounts?

Reply:
thanks for checking, yes for eu should show in first column and in non-eu should show in second column as we have derivx for non-eu